infinity-app/app.R

```diff
@@ -128,12 +171,24 @@ ui <-
       selectizeInput(
         inputId = "taxa_family",
         label = "Choose a family: (you can also select All, but it's slow so be patient)",
-        choices = "Myrtaceae",
-        selected = "Myrtaceae",
+        choices = "My",
```

Comment:
Is "My" a typo here?

Reply:
Also defaults to Asteraeceae and not All

Reply:
yes. FYI: there are actually two defaults--one very brief one before the reactive filters to the list appropriate for the polygon, and then one after that script runs.